good-fences.js

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env node
+const { goodFences } = require('./index');
```

Comment:
what is .index here? Is this a generated js bridge from napi? Is some index.node file with the actual node bindings?

Please update with a comment because I am a confused old man👴

Reply:
btw, how old are you?